spec/directives/extend/across_modules.hrx

```diff
@@ -0,0 +1,74 @@
+<===> options.yml
```

Comment:
There are already some `@extend` + `@use` specs in `spec/directives/use/extend.hrx`, so maybe these should go there as well?

Reply:
Good call, I'll move these specs there and rename them to be more consistent.